source/blog/_posts/2016-03-21-ssh-jumpserver.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Developing in the tunnel
+tags:
+    - ssh
+    - jumpserver
+    - netcat
+    - security
+github_link: blog/_posts/2016-03-21-ssh-jumpserver
+authors: [dn]
+---
+
+Securing web servers properly is a must for customers and hosters. The more complex the setup becomes the more consideration
+ is required in order to e.g. debug a customer's development system. In this blog post I will show, how you can SSH to
+ a server which is only reachable by a so called "jump server".
+
+## The setup
+The customer (green) has multiple servers in his environment. Only the *jump server* is reachable through SSH
+all other servers are protected by the firewall:
+
+![server setup](/blog/img/ssh_setup.png)
+
+The customers got my public key file beforehands and deployed it to the jump server as well as to the web server. In
+order to debug the web server, I first need to connect to the jump server:
+
+```
+daniel@localhost: ssh -A user@85.158.183.193
+```
+
+From the jump server I can then connect to the web server:
+
+
+```
+user@85.158.183.193: ssh user@10.7.0.100
+```
+
+The "magic" here comes from the `-A` ssh option. It allows me to authenticate against the web server, even without having
```

Comment:
From the Manpage:
-A "Enables forwarding of the authentication agent connection"